[RM]: convert to rst (#2746)
cntt-n/CNTT: doc/ref_model/chapters/chapter01.rst

One of major challenges holding back the more rapid and widespread adoption of virtualised/containerised network applications is when, while building or designing their virtualised services, specific infrastructure assumptions and requirements are implied, often with custom design parameters. This leaves the operators being forced to build complex integrations of various vendor/function specific silos which are incompatible with each other and might possibly have different and conflicting operating models. In addition, this makes the onboarding and conformance processes of VNFs/CNFs (coming from different vendors) hard to automate and standardise. The need for a common model across the industry to facilitate more rapid adoption is clear.

The document starts from the abstract and as it progresses it increasingly gets into more details. It follows the traditional design process where you start from core principles, progress to abstract concepts and models, then finish with operational considerations, such as security and lifecycle management.

-  **Chapter 01 - Introduction**: Overall scope of the Reference Model document including the goals and objectives of the project.

>       **Audience**: This chapter is written for a general technical audience with interest in this topic.

Reviewer: should these be literal blocks?

Author: What do you mean on literal block? Can you please explain more?

Introduction
============

Overview
--------

> The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by the Linux Foundation Networking project `Anuket <../../common/chapter00.md>`__.

Reviewer: ```suggestion
The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by the Linux Foundation Networking project `Anuket <../../common/chapter00>`__.
```

Author: At the moment the correctness of this link relies on the relative link corrector extension what needs this to use ".md" extension. We will be able to correct only those links which are in the same pr as this as the correct linking needs the .rst files.

> The Reference Model (RM) specifies a virtualisation technology agnostic (VM-based and container-based) cloud infrastructure abstraction and acts as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces required by the workloads. This document has been developed by the Linux Foundation Networking project `Anuket <../../common/chapter00.md>`__.

Reviewer 2: intersphinx?

> **Problem Statement:** Based on community consultations, including telco operators, technology suppliers, and software developers, there is a realisation that there are significant technical, operational and business challenges to the development and deployment of Virtual Network Function (VNF)/ Cloud Native Network Function (CNF) network applications related to the lack of a common cloud infrastructure platform. These include but are not limited to the following:

Reviewer: I realise that this is rst conversion but this definition needs changes. Suggest the following -- may be made in another PR: the issues is the use of "network applications" after VNF and CNF:
```suggestion
**Problem Statement:** Based on community consultations, including telco operators, technology suppliers, and software developers, there is a realisation that there are significant technical, operational, and business challenges to the development and deployment of Virtual Network Functions (VNF) and Cloud Native Network Functions (CNF) due to the lack of a common cloud infrastructure platform. These include but are not limited to the following:
```

-  **Cloud Infrastructure Software and Hardware profiles**:

   -  **Cloud Infrastructure software profiles**: These software profiles are components of the corresponding infrastructure profiles within the infrastructure profiles catalogue, and specify the host infrastructure software configurations.
   -  **Cloud Infrastructure hardware profiles**: These hardware profiles are components of the corresponding infrastructure profiles within the infrastructure profiles catalogue, and specify the host infrastructure hardware configurations.

-  **Conformance and verification**:

   -  **Conformance programs**: These define the requirements for verification and validation programs for both the cloud infrastructure and workloads.
   -  **Test framework**: This document provides input into test suites to allow conformance of cloud infrastructure and workloads.

Principles
----------

> The Reference Model specifications conform to the overall principles defined in `Anuket Principles <../../common/chapter00.md#2.0>`__.

Reviewer: needs to change to the rst version but section number may not work unless explicitly declared

Author: See my previous comment, and comments in https://github.com/cntt-n/CNTT/pull/2737#discussion_r776459859 and in https://github.com/cntt-n/CNTT/pull/2733#discussion_r776373145

Definitions/Terminology/Abbreviations
-------------------------------------

> To help guide the reader, the Reference Model `Glossary <../../common/glossary.md>`__ provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the ETSI GR NFV 003 [1] definitions. In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.

Reviewer: ```suggestion
To help guide the reader, the Reference Model `Glossary <../../common/glossary>`__ provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the ETSI GR NFV 003 [1] definitions. In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```

Author: See https://github.com/cntt-n/CNTT/pull/2746/files#r776460947

> Please refer to `Abbreviations <../../common/abbreviations.md>`__ for a full list of abbreviations used in this document.

Reviewer: ```suggestion
Please refer to `Abbreviations <../../common/abbreviations>`__ for a full list of abbreviations used in this document.
```

Author: See https://github.com/cntt-n/CNTT/pull/2746/files#r776460947

Conventions
-----------

“The key words “must”, “must not”, “required”, “shall”, “shall not”, “should”, “should not”, “recommended”, “may”, and “optional” in this document are to be interpreted as described in RFC2119 [2].”

References
----------

> Please refer to `References <../../common/references.md>`__ for a full list of references used in this document.

Reviewer: ```suggestion
Please refer to `References <../../common/references>`__ for a full list of references used in this document.
```

Author: See https://github.com/cntt-n/CNTT/pull/2746/files#r776460947
